docs/admin/projects.rst

```diff
@@ -437,12 +437,28 @@ to be escaped as ``[[]`` or ``[]]``.
 Screenshot file mask
 ++++++++++++++++++++
 
-Mask of files to allow discovery/update of screenshots, including path from vcs repository.
-It should include one "*" replacing screenshot file name.
+This feature allows the discovery and updating of screenshots through screenshot file masks, using paths from the VCS repository.
+This operates at the component level and necessitates the use of an asterisk "*" to replace the screenshot file name.
 
 Allowed formats are JPEG, PNG, APNG and GIF.
 
-For example ``docs/*.png``.
+Note: The filemask and screenshot filemask are not related. Configure them separatly.
+
+For example:
+
+Let's assume your VCS repository has a structure like this:
+
+- repository_root
+   - component_A
+      - docs
+         - image1.png
+         - image2.jpg
+
+For component_A, you want to allow discovery and updates of PNG screenshots.
+You'd set the screenshot mask for component_A as ``component_A/docs/*.png``.
+This means any PNG images under docs in component_A can be discovered and updated.
```

Comment:
So image names are never linked to translation keys AFAIU: it's a manual job to link a discovered screenshot in a component to a specific translation key? Might worth indicating it if it's the case.

Reply:
@surli Thanks for your comment. When a new screenshot is discovered in the VCS, a new screenshot object is created and linked to `component.translation` as before, no changes in the behavior there.

Please let me know if that answers your question.

Reply:
Partly yes: you confirm that there's no automatic attachment of the screenshot to a particular translation. It remains a manual work to perform the link?

Reply:
Yes.

Reply:
@surli I have updated the commit. Please let me know if it is clear now.